libraries/chain/include/graphene/chain/withdraw_permission_object.hpp

```diff
@@ -69,6 +110,23 @@ namespace graphene { namespace chain {
               ? withdrawal_limit.amount - claimed_this_period
               : 0, withdrawal_limit.asset_id );
         }
+
+       /***
+        * Get a description of the current withdrawal period
+        * @param current_time   Current time
+        * @return A description of the current period
+        */
+       withdrawal_period_descriptor current_period(fc::time_point_sec current_time) const {
```

Comment:
Fair point, although I have this object being used for code (that is not yet committed) which is related to informing a user/client software about what can still be withdrawn during any particular period.

Do you suggest that I:
(1) temporarily move this code out of the main code, and then later
(2) move this class into the main code when committing my future code?